### PR TITLE
Docs-4188 SCALE 22.02 Updated Periodic Snapshot Task Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit updates the PeriodicSnapshotTaskScreenSCALE.md article to the UI ref format with new screenshots. It updates the PeriodicSnapshotTaskSCALE.md article tutorial content to a procedure flow with new section headers and internal references.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md [hunk 1]
```diff
@@ -1,88 +1,98 @@
...
-### Naming Schemas
+![DataProtectPeriodicSnapshotTasksDetailsNoTasks](/images/SCALE/22.02/DataProtectPeriodicSnapshotTasksDetailsNoTasks.png "Periodic Snapshot Task No Tasks")
 
-The **Naming Schema** determines how automated snapshot names generate.
-A valid schema requires the *%Y* (year), *%m* (month), *%d* (day), *%H* (hour), and *%M* (minute) time strings, but you can add more identifiers to the schema too, using any identifiers from the Python [strptime function](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior).
+**Columns** displays a dropdown list of options to customize the list view. Options are **Select All**, **Recursive**, **Naming Schema**, **When**, **Frequency**, **Next Run**, **Keep snapshot for**, **VMWare Sync**, **Enabled**, **State**, and **Reset to Defaults**.
 
-{{< hint warning >}}
-For **Periodic Snapshot Tasks** used to set up a replication task with the **Replication Task** function:
+The **State** on the list view does not link to the log file or anything else, it just displays the current state of the task.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md:49]:
```suggestion
The **State** on the list view does not link to the log file or anything else. It only displays the current state of the task.
```

File: content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md [hunk 2]
```diff
@@ -1,88 +1,98 @@
...
 
-You can use custom naming schema for full backup replication tasks. If you are going to use the snapshot for an incremental replication task, use the default naming schema. Go to [Using a Custom Schema]({{< relref "TroubleshootingTips.md" >}}) for additional information.
-{{< /hint >}}
+Click the <span class="material-icons">expand_more</span> expand icon at the right of the task opens the details for the selected task.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md:51]:
```suggestion
Click the <span class="material-icons">expand_more</span> expand icon at the right of the task to open the details for the selected task.
```

File: content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md [hunk 3]
```diff
@@ -1,88 +1,98 @@
...
+**Delete** opens the delete dialog that removes the task from the system. 
+{{< /expand >}}
+## Add and Edit Periodic Snapshot Screens
 
-{{< hint warning >}}
-When referencing snapshots from a Windows computer, avoid using characters like `:` that are invalid in a Windows file path.
-Some applications limit filename or path length, and there might be limitations related to spaces and other characters.
-Always consider future uses and ensure the name given to a periodic snapshot is acceptable.
-{{< /hint >}}
+The **Add Periodic Snapshot Task** and **Edit Periodic Snapshot Task** display the some of the same settings.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md:61]:
```suggestion
The **Add Periodic Snapshot Task** and **Edit Periodic Snapshot Task** screens display some of the same settings.
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md [hunk 4]
```diff
@@ -1,88 +1,98 @@
...
-### Process
+![PeriodicSnapshotTaskWidgetNoTasks](/images/SCALE/22.02/PeriodicSnapshotTaskWidgetNoTasks.png "No Periodic Snapshot Tasks")
 
-{{< expand "Video Tutorial" "v" >}}
-This short video demonstrates adding a periodic snapshot task {{< embed-video name="scaleangelfishperiodicsnapshottasks" >}}
-{{< /expand >}} 
+**Add** opens the **[Add Periodic Snapshot Task]()** screen. 
 
-Go to **Data Protection > Periodic Snapshot Tasks** and click **Add**.
+Each task listed is a link that opens the **[Edit Periodic Snapshot Task]()** screen populated with with the settings for that task. Click on the **Description**, **Frequency** or **Next Run** column entry to open the edit task screen.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALEUIReference/DataProtection/PeriodicSnapshotTasksScreensSCALE.md:28]: I need to fix that relref that is missing

**Comment by micjohnson777** [comment]
Pulled the branch up to fix issues with missing internal links and the the feedback marked Resolved

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**